docs/src/tutorials/child-device-config-management.md

```diff
@@ -0,0 +1,327 @@
+3. On the right side, under configurations, click on the in *Step 1 Startup / Bootstrap of the child device* created config file (`config1`) in the column: `DEVICE-SUPPORTED CONFIGURATIONS` click on the config file (`config1`) and click on `Get snapshot from device`. This will trigger a config snapshot requests via MQTT.
+
+![get-config-snapshot](./images/get-config-snapshot.png)
+4. After receiving the request, the child device connector may **optionally** acknowledge the receipt of the request by sending an "executing" MQTT status message, as follows:
+
+**Topic:**
+
+`tedge/{child-d}/commands/res/config_snapshot`
+
+**Payload**:
+
+```json
+{
+    "status": "executing",
+    "type": "{config-type}",
+    "path": "/child/local/fs/path"
+}
+```
+
+Example:
+
+```console
+mosquitto_pub -h 127.0.0.1 -t "tedge/child1/commands/res/config_snapshot" -m '{"status": "executing", "path": "/home/pi/config/config1", "type": "config1"}'
+```
+
+5. Upload the requested config file to the URL received in the request via HTTP. After sending this 'executing' status message, the connector must upload the requested configuration file content to the url received in the request with an HTTP PUT request.
+
+Example:
+
+`
+curl -X PUT --data-binary @/home/pi/config/config2 http://127.0.0.1:8000/tedge/file-transfer/child1/config_snapshot/config1
+`
+
+6. Once the upload is complete, send a "successful" status message via MQTT as follows:
+
+**Topic:**
+
+`tedge/{child-d}/commands/res/config_snapshot`
+
+**Payload**:
+
+```json
+{
+    "status": "successful",
+    "type": "{config-type}",
+    "path": "/child/local/fs/path"
+}
+```
+
+Example:
+
+```
+mosquitto_pub -h localhost -t "tedge/child1/commands/res/config_snapshot" -m '{"status": "successful", "path": "/home/pi/config/config1", "type": "config1"}'
+```
+
+### Step 3 Handle config update request
+
+Performing config update is an 8-step process:
+
+1. Subscribe to, and receive config update requests via MQTT. Example: `mosquitto_sub -h 127.0.0.1 -t "tedge/child1/commands/req/config_update"`. In Cumulocity IoT, go to `device management` ->  `Management` -> `Configuration repository` -> `Add configuration snapshot` and fill in the fields in the pop-up . Upload a simple text file will be sufficient as an example. Give it the name config1.
+
+![add-config-snapshot](./images/add-config-snapshot.png)
+2. Click on `Devices` -> `All Devices`, then click on your device. Then click on  `Child devices` ->  then click on the in *Step 1 Startup / Bootstrap of the child device* defined child device (child1) and then click on `Configuration`.
+3. On the right side, under configurations, click on the config file (`config1`) created in *Step 1 Startup / Bootstrap of the child device* in the column: `DEVICE-SUPPORTED CONFIGURATIONS` click on the config file (`config1`)
+
+![add-config-snapshot-configurations](./images/add-config-snapshot-configurations.png)
```

Comment:
This same image could be shown after step. 2 of the Config Snaphsot section as well, although the name and the alt text of the image file doesn't make much sense as this image is just highlighting the **Configurations** tab.

Reply:
changed accordingly